Jenkinsfile

```diff
@@ -200,12 +200,20 @@ stage('Configure') {
 			],
 			database: [
 					new DatabaseBuildEnvironment(dbName: 'h2', mavenProfile: 'h2',
-							condition: TestCondition.BEFORE_MERGE,
+							condition: TestCondition.BEFORE_MERGE, dockerHubImage: null,
 							isDefault: true),
-					new DatabaseBuildEnvironment(dbName: 'mariadb', mavenProfile: 'ci-mariadb',
-							condition: TestCondition.AFTER_MERGE),
 					new DatabaseBuildEnvironment(dbName: 'postgresql', mavenProfile: 'ci-postgresql',
-							condition: TestCondition.AFTER_MERGE)
+							condition: TestCondition.AFTER_MERGE, dockerHubImage: "postgres:13.1"),
+					new DatabaseBuildEnvironment(dbName: 'mariadb', mavenProfile: 'ci-mariadb',
+							condition: TestCondition.AFTER_MERGE, dockerHubImage: "mariadb:10.5.8"),
+					new DatabaseBuildEnvironment(dbName: 'mysql', mavenProfile: 'ci-mysql',
+                    		condition: TestCondition.AFTER_MERGE, dockerHubImage: "mysql:8.0.22"),
+                    new DatabaseBuildEnvironment(dbName: 'db2', mavenProfile: 'ci-db2',
+                            condition: TestCondition.AFTER_MERGE, dockerHubImage: "ibmcom/db2:11.5.5.0"),
+                    new DatabaseBuildEnvironment(dbName: 'oracle', mavenProfile: 'ci-oracle',
+                            condition: TestCondition.AFTER_MERGE, dockerHubImage: "gvenzl/oracle-xe:11-slim"),
```

Comment:
What do you think of executing Oracle test for every PR build, since they are very likely to fail when we introduce a column whose name is too long?
You would just need to use `TestCondition.BEFORE_MERGE` here.
Maybe we should also add `postgresql`, for good measure. I don't think others are necessary, though.